test/unit/specs/VueFormGenerator.spec.js

```diff
@@ -10,24 +10,27 @@ let el, vm;
 function createFormGenerator(schema = {}, model = null, options = {}, multiple = false) {
 	el = document.createElement("div");		
 	// eslint-disable-next-line quotes
-	el.innerHTML = `<vue-form-generator :schema="schema" :model="model" :options="options" :multiple="multiple" v-ref:form></vue-form-generator>`;
+	el.innerHTML = `<vue-form-generator :schema="schema" :model="model" :options="options" :multiple="multiple" ref="form"></vue-form-generator>`;
 	vm = new Vue({
-		el: el,
 		data: {
 			schema,
 			model,
 			options,
 			multiple
 		}
-	});
+	}).$mount(el);
 
 	// console.log(el);
 
 	return [el, vm];
 }
 
 describe("VueFormGenerator.vue", () => {
-
+	describe.only("nothing", () => {
```

Comment:
Sorry about that, it was to test the build locally without changing too much. Normally, it is forbidden to push `.only()`. This will go away when I fix the test. I forget to put it back how it was.